### DISCUSSION TITLE
Open source LLMs as Assistant backend

### DISCUSSION THREAD

**Discussion by YurySolovyov** [zed-industries/zed#6588] [Feature Requests]
I see that initial focus is going to be on official OpenAI GPT-x based AIs, but in future do you guys consider supporting alternative open-source models such as CodeLlama and others, either officially or via plugins?

**Comment by maxbrunsfeld** [comment]
Definitely.

**Comment by JosephTLyons** [comment]
We now have Ollama support!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
